src/react-components/scene-ui.js

```diff
@@ -161,6 +178,23 @@ class SceneUI extends Component {
                   </button>
                 </WithHoverSound>
               </div>
+              {isOwner && sceneProjectId ? (
+                <a
+                  target="_blank"
+                  href={getReticulumFetchUrl(`/spoke/projects/${sceneProjectId}`)}
+                  className={styles.spokeButton}
+                >
+                  <FontAwesomeIcon icon={faPencilAlt} />
+                  <FormattedMessage id="scene.edit_button" />
+                </a>
+              ) : (
```

Comment:
I'm still getting my stack up to test everything. What happens when you are not logged in and click the remix scene button?

Reply:
You won't get the remix button if you are not logged in since we wouldn't detect that you own the scene. That said we do need to handle that for the "Copy to My Scenes" button. I think I will just have the button hidden for now, good catch.